ONSdigital/eq-survey-runner
# Fix relationship description bug

### What is the context of this PR?
To fix the bug in [this](https://trello.com/c/MF53AIfe/3277-relationships-matrix-not-showing-descriptions-for-person-2-onwards-s) Trello card

### How to review 
- Add 3 people to the household
- Go through relationships section
- Check that radio descriptions are displayed for every person
